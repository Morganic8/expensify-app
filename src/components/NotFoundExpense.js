import React from 'react';
import {Link} from 'react-router-dom';


const NotFoundExpense = () => (
    <div>
        404 - <Link to="/"> Go Home</Link>
    </div>
);

export default NotFoundExpense;