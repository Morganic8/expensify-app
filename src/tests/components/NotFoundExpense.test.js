import React from 'react';
import { shallow } from 'enzyme';
import NotFoundExpense from '../../components/NotFoundExpense';


test('Should render Not found expense page', () => {
    const wrapper = shallow(<NotFoundExpense />);
    expect(wrapper).toMatchSnapshot();
});