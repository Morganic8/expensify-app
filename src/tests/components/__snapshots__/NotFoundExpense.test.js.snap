// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Not found expense page 1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/"
  >
     Go Home
  </Link>
</div>
`;
